README.md

```diff
@@ -1,18 +1,24 @@
 # Journey_Lua_Modder
-A python GUI application aiming to make Lua injection to the game "Journey" easier.
+A python GUI application aiming to make Lua injection to the game "Journey" easier. 
 
-How to install:
+Also works in the game "Flower" with minor changes. (do not copy Tick.lua from here, instead just put the function details in the existing file)
```

Comment:
I'm not deciding to make this "that game Lua modder" thing yet, but you can always just clone mine and make a flower Lua modder based on that.